gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineIdea.groovy

```diff
@@ -179,14 +182,14 @@ class BaselineIdea extends AbstractBaselinePlugin {
     /**
      * Extracts copyright headers from Baseline directory and adds them to Idea project XML node.
      */
-    private void addCopyright(node) {
-        def copyrightManager = node.component.find { it.'@name' == 'CopyrightManager' }
+    private void addCopyright(Node node) {
+        Node copyrightManager = node.component.find {it.'@name' == 'CopyrightManager'}
         def copyrightDir = Paths.get("${configDir}/copyright/")
         def copyrightFiles = getCopyrightFiles(copyrightDir)
-        copyrightFiles.each { File file ->
+        copyrightFiles.each {File file ->
```

Comment:
these whitespace changes are intellij helpfully reformatting things. Given that we don't have a palantir-groovy-format (😂), I couldn't be bothered to revert em all piece by piece